src/site/content/en/fast/choose-the-right-image-format/index.md

```diff
@@ -186,17 +192,25 @@ In terms of older image formats, consider the following:
     * What about GIF? GIF limits the color palette to at most 256 colors,
       and creates significantly larger file sizes than `<video>` elements. See
       [Replace animated GIFs with video](/replace-gifs-with-videos/).
-1. **Do you need to preserve fine detail with highest resolution? Use PNG.**
+1. **Do you need to preserve fine detail with highest resolution? Use PNG or lossless WebP, or lossless AVIF.**
```

Comment:
I think lossless [AVIF is pretty poor](https://twitter.com/jaffathecake/status/1540710471333867520?s=20&t=0kOAav6dNZWALveTYHBPpQ). I'd remove this recommendation from this section.

```suggestion
1. **Do you need to preserve fine detail with highest resolution? Use PNG or lossless WebP.**
```